pkg/reconciler/sinkbinding/controller.go

```diff
@@ -80,8 +80,8 @@ func NewController(
 	dc := dynamicclient.Get(ctx)
 	psInformerFactory := podspecable.Get(ctx)
 	namespaceInformer := namespace.Get(ctx)
-	serviceaccountInformer := serviceaccountinformer.Get(ctx)
-	secretInformer := secretinformer.Get(ctx)
+	oidcServiceaccountInformer := serviceaccountinformer.Get(ctx)
```

Comment:
did we just rename it for a better name?

Reply:
Yeah the name was changed later, so when I did the cherry-pick it seems to have included that as well

Reply:
ok, I see is now same name as on main